en/docs/Administer/ProductConfigurations/changing-the-default-transport.md

```diff
@@ -1,34 +1,29 @@
 # Changing the Default Transport
 
-On the back end, APIs are Apache Synapse configurations that WSO2 API Manager accesses through a transport. The default API Manager transport is the PassThrough transport, but you can configure a different default transport in your `axis2.xml` file. For example, to use the HTTP-NIO transport as the default, go to the `<APIM_HOME>/repository/conf/axis2` folder, open the `axis2.xml` file, and then in the "Transport Ins" and "Transport Outs" sections, comment out the PassThrough configurations and uncomment the configurations for the HTTP-NIO transport.
+On the back end, APIs are Apache Synapse configurations that WSO2 API Manager accesses through a transport. The default API Manager transport is the PassThrough transport, but you can configure a different default transport in your `deployment.toml` file.
 
 !!! info
-WSO2 products do not use the HTTP/S servlet transport configurations that are in `axis2.xml` file. Instead, they use Tomcat-level servlet transports, which are used by the management console in `<PRODUCT_HOME>/repository/conf/tomcat/catalina-server.xml` file .
+    WSO2 products do not use the HTTP/S servlet transport configurations that are in `axis2.xml` file. Instead, they use Tomcat-level servlet transports, which are used by the management console in `<PRODUCT_HOME>/repository/conf/tomcat/catalina-server.xml` file .
```

Comment:
Shall we customize this as this doc is only for APIM.


[1]WSO2 products do not use the HTTP/S servlet transport configurations that are in `axis2.xml` file. Instead, they use Tomcat-level servlet transports, which are used by the management console in `<PRODUCT_HOME>/repository/conf/tomcat/catalina-server.xml` file .

[2] WSO2 API Manager do not use the HTTP/S servlet transport protocol configurations that are in the `axis2.xml` file. Instead, WSO2 API Manager's Management Console uses the Tomcat-level servlet transport protocols that are available in the `<API-M_HOME>/repository/conf/tomcat/catalina-server.xml` file.

Reply:
fixed with https://github.com/wso2/docs-apim/pull/540/commits/66d4d47c3b663dccc4ce708bc0ae2f9cf1e0ddff